deps/userns_sandbox.c

```diff
@@ -431,7 +432,10 @@ static void mount_the_world(const char * root_dir,
     // Create tmpfs to store ephemeral changes.  These changes are lost once
     // the `tmpfs` is unmounted, which occurs when all processes within the
     // namespace exit and the mount namespace is destroyed.
-    check(0 == mount("tmpfs", "/proc", "tmpfs", 0, "size=1G"));
+    char * options = NULL;
+    check(0 < asprintf(&options, "size=%s", tmpfs_size));
```

Comment:
I would prefer we use `snprintf()` with an `options` that is statically allocated, say 32 bytes large.  No need to allocate and free for this.

Reply:
Done!